tests/prebuild-test.js

```diff
@@ -24,6 +24,9 @@ describe('prebuild', function() {
         let result = prebuildRollUp.preBuild(addonPath);
         return result.then(() => {
             expect(fs.readdirSync(preBuildPath)).to.deep.equal(['addon','vendor']);
+            fs.lstatSync(path.join(preBuildPath,'addon'), function(err, stats) {
```

Comment:
I dont think `lstatSync` takes a second argument, I believe it is never invoked.

I believe the following, should do the trick
```js
let stat =  fs.lstatSync(path.join(preBuildPath,'addon');
expect(stats.isSymbolicLink()).to.be.false;
```